resources/CICD/fastlane/Fastfile

```diff
@@ -0,0 +1,198 @@
+# frozen_string_literal: true
+
+require './Constants/Constants'
+require './Managers/VersioningManager'
+require './Managers/BuildManager'
+require './Managers/SymbolManager'
+require './Managers/DistributionManager'
+
+versioning_manager = VersioningManager.new(
+  fastlane: self,
+  project_path: Constants.PROJECT_PATH,
+  main_target_name: Constants.MAIN_TARGET_NAME
+)
+
+symbol_manager = SymbolManager.new(
```

Comment:
We have faced an issue related to this instance, because of `versioning_manager.version_number` and `versioning_manager.build_number`. Let's init it when we need it.